plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -311,6 +318,13 @@ private static function newly_installed() {
 			do_action_deprecated( 'woocommerce_admin_newly_installed', array(), '6.5.0', 'woocommerce_newly_installed' );
 
 			update_option( self::NEWLY_INSTALLED_OPTION, 'no' );
+
+			/**
+			 * This option is used to track the initial version of WooCommerce that was installed.
+			 *
+			 * @since 9.2.0
+			 */
+			add_option( self::INITIAL_INSTALL_VERSION_OPTION, WC()->version );
```

Comment:
I think we should make this option non-autoload since we may not require it for every request.

Reply:
Another great suggestion, done in https://github.com/woocommerce/woocommerce/pull/49139/commits/ac37ede0eef14e318b3cb825f89ada3c87845b55